### PR TITLE
Add all required elements for case studys landing page + image with q…

### PR DESCRIPTION
**Pull request by bitterdev**
…uote block type

### CODE REVIEW

File: public/packages/concrete_cms_marketing/blocks/page_list/templates/case_study_list/view.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    <div class="row">
+        <div class="col">
+            <h2 class="">
+                <?php echo $pageListTitle; ?>
+            </h2>
+        </div>
+    </div>
+
+    <div class="row">
+        <?php foreach ($pages as $page) { ?>
```
Comment (aembler) [inline public/packages/concrete_cms_marketing/blocks/page_list/templates/case_study_list/view.php:43]: This is actually gonna be a summary template so we can use the same markup easily in multiple places. See the video attached to the ticket.

File: public/packages/concrete_cms_marketing/src/PortlandLabs/ConcreteCmsMarketing/Provider/ServiceProvider.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+use PortlandLabs\ConcreteCmsMarketing\Search\Pagination\View\Manager;
+use PortlandLabs\ConcreteCmsMarketing\Search\Pagination\View\ManagerServiceProvider as PaginationManager;
+use PortlandLabs\ConcreteCmsMarketing\Search\Pagination\View\PagerManager;
+
+class ServiceProvider extends Provider
+{
+
+    public function register()
+    {
+        $this->app['manager/view/pagination'] = $this->app->share(function ($app) {
```
Comment (aembler) [inline public/packages/concrete_cms_marketing/src/PortlandLabs/ConcreteCmsMarketing/Provider/ServiceProvider.php:23]: I'm not sure we need this here? Don't we have custom paging within the Concrete CMS theme? We don't want different pagination in the theme vs in the marketing site. All the pagination should be defined within the concrete cms theme package.

File: public/packages/concrete_cms_marketing/src/PortlandLabs/ConcreteCmsMarketing/Search/Pagination/View/SimplePaginationTemplate.php [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+ *
+ * @copyright  (C) 2021 Portland Labs (https://www.portlandlabs.com)
+ * @author     Fabian Bitter (fabian@bitter.de)
+ */
+
+namespace PortlandLabs\ConcreteCmsMarketing\Search\Pagination\View;
+
+use Pagerfanta\View\Template\Template;
+
+class SimplePaginationTemplate extends Template
```
Comment (aembler) [inline public/packages/concrete_cms_marketing/src/PortlandLabs/ConcreteCmsMarketing/Search/Pagination/View/SimplePaginationTemplate.php:14]: I think we want to move all this into the theme class.

**Review by aembler — CHANGES_REQUESTED** [review]

File: public/packages/concrete_cms_marketing/blocks/page_list/templates/case_study_list/view.php (hunk 1 above)
Comment (bitterdev) [reply public/packages/concrete_cms_marketing/blocks/page_list/templates/case_study_list/view.php:43]: @aembler Hi Andrew. I have seen the video and already replied on jira. The problem is that the summary feature doesn't work well with 3rd party packages. Please check my comments on jira. You can work anyway with that custom template. We can replace it later and switch to summary templates without touching the content.

File: public/packages/concrete_cms_marketing/src/PortlandLabs/ConcreteCmsMarketing/Provider/ServiceProvider.php (hunk 2 above)
Comment (bitterdev) [reply public/packages/concrete_cms_marketing/src/PortlandLabs/ConcreteCmsMarketing/Provider/ServiceProvider.php:23]: In Gary layouts is another pagination used for it. Therefore i have created a custom one. I will move it to the theme repository tomorrow.

File: public/packages/concrete_cms_marketing/src/PortlandLabs/ConcreteCmsMarketing/Search/Pagination/View/SimplePaginationTemplate.php (hunk 3 above)
Comment (bitterdev) [reply public/packages/concrete_cms_marketing/src/PortlandLabs/ConcreteCmsMarketing/Search/Pagination/View/SimplePaginationTemplate.php:14]: Yes will do so tomorrow :)

File: public/packages/concrete_cms_marketing/blocks/page_list/templates/case_study_list/view.php (hunk 1 above)
Comment (bitterdev) [reply public/packages/concrete_cms_marketing/blocks/page_list/templates/case_study_list/view.php:34]: @aembler Okay i have fixed it. The updated PR takes use of summary templates. Pagination are now move to the theme package.

**Comment by aembler** [comment]
I think we no longer need this pull request. If I'm wrong please reopen.

**Pull request closed without merging**